openshift/backup/backup-deploy.json

```diff
@@ -0,0 +1,452 @@
+                        "key": "${SSO_PGSQL_MASTER_USER_KEY_NAME}"
+                      }
+                    }
+                  },
+                  {
+                    "name": "SSO_PGSQL_MASTER_PASSWORD",
+                    "valueFrom": {
+                      "secretKeyRef": {
+                        "name": "${DATABASE_DEPLOYMENT_NAME}",
+                        "key": "${SSO_PGSQL_MASTER_PASSWORD_KEY_NAME}"
+                      }
+                    }
+                  },
+                  {
+                    "name": "WEBHOOK_URL",
+                    "value": "${WEBHOOK_URL}"
+                  },
+                  {
+                    "name": "ENVIRONMENT_FRIENDLY_NAME",
+                    "value": "${ENVIRONMENT_FRIENDLY_NAME}"
+                  },
+                  {
+                    "name": "ENVIRONMENT_NAME",
+                    "value": "${ENVIRONMENT_NAME}"
+                  }
+                ],
+                "resources": {
+                  "requests": {
+                    "cpu": "${CPU_REQUEST}",
+                    "memory": "${MEMORY_REQUEST}"
+                  },
+                  "limits": {
+                    "cpu": "${CPU_LIMIT}",
+                    "memory": "${MEMORY_LIMIT}"
+                  }
+                },
+                "volumeMounts": [
+                  {
+                    "name": "${BACKUP_VOLUME_NAME}",
+                    "mountPath": "${BACKUP_DIR}"
+                  },
+                  {
+                    "name": "${VERIFICATION_VOLUME_NAME}",
+                    "mountPath": "${VERIFICATION_VOLUME_MOUNT_PATH}"
+                  },
+                  {
+                    "name": "${NAME}-config-volume",
+                    "mountPath": "${CONFIG_MOUNT_PATH}${CONFIG_FILE_NAME}",
+                    "subPath": "${CONFIG_FILE_NAME}"
+                  }
+                ]
+              }
+            ]
+          }
+        }
+      }
+    }
+  ],
+  "parameters": [
+    {
+      "name": "NAME",
+      "displayName": "Name",
+      "description": "The name assigned to all of the resources defined in this template.",
+      "required": true,
+      "value": "backup"
+    },
+    {
+      "name": "SOURCE_IMAGE_NAME",
+      "displayName": "Source Image Name",
+      "description": "The name of the image to use for this resource.",
+      "required": true,
+      "value": "backup"
+    },
+    {
+      "name": "IMAGE_NAMESPACE",
+      "displayName": "Image Namespace",
+      "description": "The namespace of the OpenShift project containing the imagestream for the application.",
+      "required": true,
+      "value": "backup-container"
+    },
+    {
+      "name": "TAG_NAME",
+      "displayName": "Environment TAG name",
+      "description": "The TAG name for this environment, e.g., dev, test, prod",
+      "required": true,
+      "value": "dev"
+    },
+    {
+      "name": "DATABASE_NAME",
```

Comment:
DATABASE_NAME is not required when using a backup.conf.  You can remove the parameter and the associated environment variable.